crates/runtime/src/component/dataset/acceleration.rs

```diff
@@ -358,9 +358,9 @@ impl TryFrom<spicepod_acceleration::Acceleration> for Acceleration {
 
         let mut params = acceleration.params.clone();
 
-        let disable_query_push_down = match params
+        let disable_federation = match params
             .as_mut()
-            .and_then(|x| x.data.remove("disable_query_push_down"))
```

Comment:
`disable_query_push_down` is not documented param we added here: https://github.com/spiceai/spiceai/pull/2327